libs/payments/cart/src/lib/cart.service.ts

```diff
@@ -162,7 +162,7 @@ export class CartService {
           if (invoice) {
             switch (invoice.status) {
               case 'draft':
-                await this.invoiceManager.delete(invoice.id);
+                await this.invoiceManager.disableAutoAdvance(invoice.id);
```

Comment:
This would leave the invoice hanging in draft. Is that the desired outcome?

Do you happen to have any Stripe docs that might provide some guidance on how to handle this situation appropriately?

Reply:
I had a quick look myself, and wasn't able to find anything definitive, but finalizing the invoice and then marking as `void` seems like a good approach?

Reply:
This was my first thought as well! Back-to-back awaited calls to first finalize, and then void, the invoice, through a "cannot void a non finalized invoice" error. I didn't dive too deep into it, and it's possible this is caused by a caching issue on our end.

If it's preferable I can look more into this flow! I'm currently AFK so debugging is hard at this instant :)

Reply:
ok changes are up, and everything seems to be working as expected!